Makefile

```diff
@@ -115,7 +115,7 @@ cover:
 .PHONY: cover
 
 dockerbuild-go:
-	docker build -t $(DOCKER_IMAGE_NAME):$(BRANCH) .
+	docker build --no-cache -t $(DOCKER_IMAGE_NAME):$(BRANCH) .
```

Comment:
does this really help us? it will slow down the building process. didn't "pullAlways" solve our problem?

Reply:
Basically you are right it will slow down the build. On travis it will have no impact.
On the local it will ensure that you have the most recent code.
I think it is better that we know exactly what we are building rather than try to save time.
If we see that this is something that blocks us we can revert it later